addons/web/static/src/js/core/dom.js

```diff
@@ -539,28 +539,32 @@ var dom = {
     /**
      * @param {HTMLElement} el - the element to stroll to (limitation: if the
      *      element is using a fixed position, this function cannot work except
-     *      if is the header (with the "top" id) or the footer (with the
-     *      "bottom" id) for which exceptions have been made)
+     *      if is the header (el is then a string set to '#top') or the footer
+     *      (el is then a string set to '#bottom') for which exceptions have
+     *      been made)
```

Comment:
I imagined `scrollTo('top')` and not `scrollTo('#top')` but it's actually nice the way you went as it allows to use the function by sending any selector (not only top and bottom) :+1: However, the js doc has to be adapted more than what you did: the param type is not good anymore. Your change "(el is then a string set to '#top')" is also wrong as you can receive the header element directly and it should still work as before too.

Reply:
Concerning the js doc, the parameter type for el is HTML element or string. I was wondering if it is implicit that for an HTML element, the function is waiting for an element that is obligatorily in the DOM or should we explicitly write it ?